float/e_rem_pio2f.c

```diff
@@ -77,7 +77,10 @@ pio2_3t =  6.1232342629e-17f; /* 0x248d3132 */
 #ifdef _MSC_VER
 static __forceinline
 #else
-static __inline __always_inline
```

Comment:
A better change would be to replace this line with `static __inline __attribute__((always_inline))` instead which should be supported for both clang and gcc.